network/src/ledger.rs

```diff
@@ -510,12 +513,17 @@ impl<N: Network, E: Environment> Ledger<N, E> {
                 // Attempt to add the unconfirmed block as the next block in the canonical chain.
                 false => match self.canon.add_next_block(&unconfirmed_block) {
                     Ok(()) => {
+                        // Allocation necessary if prometheus is enabled.
```

Comment:
```suggestion
```
since there is no functional difference either way, no need for the comment; also, I think it was meant to say "Assignment" instead